.github/workflows/lint.yml

```diff
@@ -1,9 +1,7 @@
 name: Lint
 
-on: # yamllint disable-line rule:truthy
-  pull_request: {}
-  push:
-    branches: [main]
```

Comment:
similarly I think it's fine to run the `linting` workflow when we merge PR's into the `main` branch.